R/nicenumericcuts.R

```diff
@@ -0,0 +1,657 @@
+#'      and lower ends of the range should be open-ended or should contain
+#'      both end points of the interval. E.g. if \code{TRUE} then interval
+#'      (0, 17] would be "Less than 18", otherwise it would be "0 to 17".
+#' @param label.style A character indicating the style of labels to use
+#'      for the new factor levels. Options are \code{tidy.labels}, 
+#'      \code{inequality.notation}, \code{interval.notation}, and
+#'      \code{percentiles}, with the latter option only being applicable
+#'      if \code{method} is also \code{percentiles}.
+#' @param number.prefix A character to be appended before numbers in new
+#'      factor labels.
+#' @param number.suffix A character to be after before numbers in new
+#'      factor labels.
+#' @param open.bottom.string A character indicating text to be placed at the
+#'      beginning of open-ended labels at the start of the range for open
+#'      intervals.
+#' @param closed.bottom.string A character indicating text to be placed at the
+#'      end of open-ended labels at the start of the range for closed
+#'      intervals.
+#' @param open.top.string A character indicating text to be placed at the
+#'      beginning of open-ended labels at the end of the range for open
+#'      intervals.
+#' @param closed.top.string A character indicating text to be placed at the
+#'      end of open-ended labels at the end of the range for closed
+#'      intervals.
+#' @param equal.intervals.start A numeric value indicating the start of the
+#'      range when using \code{method} of \code{equal.width}.
+#' @param equal.intervals.end A numeric value indicating the end of the
+#'      range when using \code{method} of \code{equal.width}.
+#' @param custom.breaks A character containing a comma-seprated list of 
+#'      numeric values to be used as custom break points when the 
+#'      \code{method} is \code{Custom}.
+#' @param percents A single numeric value, or a character containing a comma
+#'      -separated list of numeric values to be used when the \code{method}
+#'      of \code{percentiles} is used. Values should be between 0 and 100.
+#' @param quantile.type An interger between 1 and 9 to be passed to
+#'      \code{quantile} which determines the algorithm for creating quantiles.
+#' @param factors.use.labels A logical value indicating whether numeric
+#'      information should be extracted from factor labels. If \code{FALSE}
+#'      the function will instead try to extract the underlying numerice
+#'      values from Q/Displayr.
+#' @param grouping.mark A character to be used as the thousands-grouping
+#'      mark when inferring numeric information from factor labels. 
+#' @param decimals.mark A character to be used as the decimals-grouping
+#'      mark when inferring numeric information from factor labels. 
+#'
+#' @importFrom flipU ConvertCommaSeparatedStringToVector
+#' @importFrom plyr mapvalues
+#' @importFrom stats var quantile
```

Comment:
You may want to add `#' @return` to document the returned object and `#' @examples`  to provide example usage.